package.json

```diff
@@ -360,17 +360,14 @@
     "webpack-bundle-analyzer": "3.9.0"
   },
   "resolutions": {
-    "@graphql-tools/git-loader": "6.2.6",
-    "@testing-library/dom": "7.29.1",
     "axios": "0.21.1",
     "immer": "8.0.1",
     "marked": "2.0.0",
     "node-forge": "0.10.0",
     "object-path": "0.11.5",
     "serialize-javascript": "4.0.0",
-    "webpack": "4.44.1",
     "xml-crypto": "2.0.0",
     "prismjs": "1.23.0",
-    "urijs": "1.19.6"
```

Comment:
Yes, this is the only change related to the audit issue. The other changes were made because those resolutions are now redundant. I'm adding the webpack one back, do you want me to add the other ones back as well?

Reply:
Webpack is enough.